packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/ImageMenu/index.tsx

```diff
@@ -82,6 +83,12 @@ export default (props: FloatingImageMenu) => {
     setAlt(node.attrs.alt)
   }, [selectedImage.node])
 
+  useEffect(() => {
+    setTimeout(() => {
+      if (inputRef.current) inputRef.current.focus()
+    })
+  }, [inputRef])
```

Comment:
The `inputRef` object never actually changes. If you want this to re-run every time it's value changes then it would look like this:

```ts
  }, [inputRef.current])
```

If you only want it to run the once then it would be:

```ts
  }, [])
```

Reply:
I think this conditions can be improved, I will check.

Reply:
I tested this and found that current condition is working well.

PR is good to be merged.